### PR TITLE
Section.py: Remove redundant lines

### PR DESCRIPTION
**Pull request by yashLadha**
Sectio.py: Removed redundant code

Removed redundant return statements from nested if Body

Fixes Issue #3822

### CODE REVIEW

**Comment by yashLadha** [comment]
Please review @adtac and can you tell me why my build is failing

File: coalib/settings/Section.py [hunk 1]
```diff
@@ -209,8 +209,6 @@ def __getitem__(self, item, ignore_defaults=False):
                 res.value = self.defaults[key]._value + ', ' + res._value
                 res.to_append = False
                 return res
-            res.to_append = False
-            return res
```
Comment (satwikkansal) [inline coalib/settings/Section.py]: Removed the wrong lines :)

**Comment by satwikkansal** [comment]
Also your commit message needs work. Please follow the commit guidelines at http://coala.io/commit

**Comment by yashLadha** [comment]
Review PR @satwikkansal

**Comment by Siddhartha1234** [comment]
@yashLadha rebase your changes to a single commit and it will be ready.

**Comment by adtac** [comment]
@yashLadha please don't directly ping someone for a review if it's not really particularly relevant to them or if you haven't discussed the PR/issue with them on gitter or elsewhere.

And please take a look at http://coala.io/commit and follow the guidelines. It looks like jayvdb commented on this in an earlier PR of yours.

**Comment by yashLadha** [comment]
Squashed The commits

**Comment by Siddhartha1234** [comment]
ack 2d89495

**Comment by Techievena** [comment]
unack 2d89495

**Comment by yashLadha** [comment]
What can be Improved in this?

**Comment by yashLadha** [comment]
Can you suggest some Improvement?

**Comment by Techievena** [comment]
Your PR has no body or link to issue. Kindly go through https://coala.io/commit to write a proper commit message

**Comment by yashLadha** [comment]
How about it?

**Comment by Techievena** [comment]
about what? @yashLadha

**Comment by yashLadha** [comment]
The PR.  I had added Comment to it which references to the Issue.

**Comment by tabias** [comment]
changes look good :+1: 

commit message shortlog is ok, closing link is ok, description is missing

**Comment by jayvdb** [comment]
@yashLadha, in future, please do not do development work on the "master" branch.

**Comment by yashLadha** [comment]
Okk , on which branch should I work.

**Comment by tabias** [comment]
ack 18ef06a

**Comment by yashLadha** [comment]
Created changes @jayvdb

**Comment by Udayan12167** [comment]
@yashLadha Just reiterating to not tag people in your comments when you have made the changes. If the PR is marked `process/pending review` someone will take a look at it.  If you tag people it unnecessarily sends them a notification. While you wait for a review I would request you to also help us reviewing other PRs as we are experiencing heavy traffic these days. Thank you :).

**Comment by Udayan12167** [comment]
ack d25a624

**Comment by supergr35** [comment]
Please rebase :)

**Comment by yashLadha** [comment]
What exactly i need to do?

**Comment by fneu** [comment]
sorry we don't do merge commits :/

rather:

- make sure you pulled/fetched latest master
- check out your branch and run `git rebase master`
- sometimes `git rebase --interactive` comes in handy instead, read up on those commands :)

**Comment by yashLadha** [comment]
Rebased Properly

**Comment by fneu** [comment]
reack https://github.com/coala/coala/pull/3838/commits/ea674e004ad4d5c8c8276c970d36f8582f7ff2cf

**Comment by fneu** [comment]
@rultor merge

**Pull request merged**